addon/ng2/commands/set.ts

```diff
@@ -0,0 +1,25 @@
+import * as Command from 'ember-cli/lib/models/command';
+import {CliConfig} from '../models/config';
+
+
+const SetCommand = Command.extend({
+  name: 'set',
+  description: 'Set a value in the configuration.',
+  works: 'outsideProject',
+
+  availableOptions: [
+    { name: 'global', type: Boolean, default: false, aliases: ['g'] },
+  ],
+
+  run: function (commandOptions, rawArgs): Promise<void> {
+    return new Promise(resolve => {
+      const config = new CliConfig();
+      config.set(rawArgs[0], rawArgs[1], commandOptions.force);
+      config.save();
+      resolve();
+    });
+  }
+});
+
+module.exports = SetCommand;
+module.exports.overrideCore = true;
```

Comment:
You don't need to override this command as it doesn't exists in `ember-cli`. That also stands for `get`.